src/board.py

```diff
@@ -1,14 +1,18 @@
-from pieces import Pawn, Bishop, Knight, Rook, Queen, King
+"""Implements a chessboard."""
+from .pieces import Piece, Pawn, Bishop, Knight, Rook, Queen, King
```

Comment:
If this works, I prefer relative path-ing too.

Reply:
It does if it is run as a module.